### PR TITLE
Add MQTTX

### PR DESCRIPTION
**Pull request by ysfscream**
**Description**: 

MQTTX is a MQTT desktop client open sourced by EMQ, It allows users to quickly test the MQTT/MQTTS connection, publish and subscribe to MQTT messages.
It's built with `Electron` & `Vue` & `TypeScript`. You can learn how to use electron in Vue and typescript.

**By submitting this pull request, I promise I have read the [contributing guidelines](https://github.com/sindresorhus/awesome-electron/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

### CODE REVIEW

**Comment by sindresorhus** [comment]
Your readme assumes that everyone ending up there knows what MQTTX is. I think you need to improve that. Also the description here could be better.

**Comment by ysfscream** [comment]
@sindresorhus Hi, thank you for your reply. I think there should be no problem with my readme, for which there are new manuals introduced. https://github.com/emqx/MQTTX/blob/master/docs/manual.md#Introduction

File: readme.md [hunk 1]
```diff
@@ -177,6 +177,7 @@ Made with Electron.
 - [ExifCleaner](https://github.com/szTheory/exifcleaner) - Clean image metadata with drag and drop.
 - [massCode](https://github.com/antonreshetov/massCode) - Code snippet manager for developers.
 - [Swifty](https://github.com/swiftyapp/swifty) - Password manager.
+- [MQTTX](https://github.com/emqx/MQTTX) - An elegant cross-platform MQTT 5.0 desktop client
```
Comment (sindresorhus) [inline readme.md]: The description should describe what MQTT is.

**Comment by sindresorhus** [comment]
Your manual does not help resolve my comment and the manual is not even linked from the readme. The readme should clearly explain what the project is in layman's terms. It currently fails at that.

**Comment by ysfscream** [comment]
@sindresorhus OK, thanks. After thinking about it, I will improve my readme.

**Comment by sindresorhus** [comment]
Much better 👌

**Comment by ysfscream** [comment]
@sindresorhus Thanks!
